media/CfgEditor/cfgeditor.html

```diff
@@ -247,6 +247,11 @@
                     <vscode-text-field id="EdgeTPUIntermediateTensorsInputArrays" startIcon="true" size="50" placeholder="semi-colon separated input names">
                         Intermediate_tensors
                     </vscode-text-field> 
+                    <span class="codicon codicon-question" style="cursor: pointer">
+                        <span class="help">
+                            This option cannot be with 'search delegate'
+                        </span>
+                    </span>
```

Comment:
Since we don't support the help option, We should delete these codes.

Reply:
Oh, I  mentioned this code about you but I fotgot that this is not help option, only help message.
I confused, Sorry T_T